### PR TITLE
Add submodules to CI testing

### PR DESCRIPTION
**Pull request by dannylamb**
**Github issue**: https://github.com/Islandora/documentation/issues/1390

# What does this Pull Request do?

Adds all the submodules to CI testing with Travis.  We were only running the breadcrumbs submodule before.

# What's new?
Testing, and fixes to make the tests work.

# How should this be tested?

Confirm that Travis has been appeased.

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ script:
   - $SCRIPT_DIR/travis_scripts.sh
   - $SCRIPT_DIR/run-tests.sh "islandora"
   - $SCRIPT_DIR/run-tests.sh "islandora_breadcrumbs"
+  - $SCRIPT_DIR/run-tests.sh "islandora_image
```
Comment (whikloj) [inline .travis.yml]: Missing closing `"`

**Comment by dannylamb** [comment]
Having just diagnosed the errors for islandora_video, it looks like all the failures from the new modules are related to schema errors.  I can fix it pretty easily.

**Comment by whikloj** [comment]
@dannylamb you doing that here? Also there is a coder_sniffer error.
https://travis-ci.org/Islandora/islandora/jobs/626770096#L2376

**Comment by dannylamb** [comment]
Ima doin' it.  I can put it here or with the other PR I have that fixes tests.

**Comment by whikloj** [comment]
I merged https://github.com/Islandora/islandora/pull/747, if that's the one you mean.

**Comment by dannylamb** [comment]
@whikloj You sir are both a :man_in_tuxedo:  and a :man_teacher:

**Comment by dannylamb** [comment]
Gonna have a long wait for Travis here, but I think I got it.

**Pull request merged**